tests/packagedcode/data/go/kingpin/output.expected.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "type": "golang",
+    "namespace": null,
+    "name": "kingpin",
+    "version": null,
+    "qualifiers": {},
+    "subpath": null,
+    "primary_language": "Go",
+    "description": null,
+    "release_date": null,
+    "parties": [],
+    "keywords": [],
+    "homepage_url": "https://pkg.go.dev/github.com/alecthomas/kingpin",
```

Comment:
`https://pkg.go.dev/github.com/alecthomas/kingpin` is the `repository_homepage_url` alright... but would the homepage be instead `https://github.com/alecthomas/kingpin` ? You may want to sync up with @TG1999 as he is working on purl <-> urls too

Reply:
@pombredanne It is not always fixed the module section contains the github url. It can be like `google.golang.org/grpc` too. So I think it will be better that we use `https://pkg.go.dev/github.com/alecthomas/kingpin` type here :)

I have ping @TG1999 here too for his comments :)

Reply:
In all cases, `https://pkg.go.dev/github.com/alecthomas/kingpin` is a `repository_homepage_url`
For the cases where you can infer a good `homepage_url` do it, such as here: `https://github.com/alecthomas/kingpin`

Do not hesitate to collect ALL the package listed at  https://index.golang.org/ to devise smarter rules informed from the actual data?

Reply:
@pombredanne 
> Do not hesitate to collect ALL the package listed at https://index.golang.org/ to devise smarter rules informed from the actual data?

Can you explain more about this line, what you should I actually so with this site?? https://index.golang.org/ does not contain all the package list. maximum limit is 2000